### ISSUE TITLE
ScrollViewer headers hidden from accessibility tools when used in ListView control template

### ISSUE THREAD

**Issue by RBrid** [microsoft/microsoft-ui-xaml#4698]
When the ScrollViewer inside the ListView control template uses headers (i.e. ScrollViewer.TopLeftHeader, LeftHeader, TopHeader), those headers are not discoverable through automation tools like Accessibility Insights for Windows.

Markup:
```
    <Grid x:Name="sp" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
        <Grid.Resources>
            <Style x:Key="CustomListViewStyle" TargetType="ListView">
                <Setter Property="IsTabStop" Value="False" />
                <Setter Property="TabNavigation" Value="Once" />
                <Setter Property="IsSwipeEnabled" Value="True" />
                <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Disabled" />
                <Setter Property="ScrollViewer.VerticalScrollBarVisibility" Value="Auto" />
                <Setter Property="ScrollViewer.HorizontalScrollMode" Value="Disabled" />
                <Setter Property="ScrollViewer.IsHorizontalRailEnabled" Value="False" />
                <Setter Property="ScrollViewer.VerticalScrollMode" Value="Enabled" />
                <Setter Property="ScrollViewer.IsVerticalRailEnabled" Value="True" />
                <Setter Property="ScrollViewer.ZoomMode" Value="Disabled" />
                <Setter Property="ScrollViewer.IsDeferredScrollingEnabled" Value="False" />
                <Setter Property="ScrollViewer.BringIntoViewOnFocusChange" Value="True" />
                <Setter Property="UseSystemFocusVisuals" Value="True" />
                <Setter Property="ItemsPanel">
                    <Setter.Value>
                        <ItemsPanelTemplate>
                            <StackPanel x:Name="StackPanel" Orientation="Vertical"/>
                        </ItemsPanelTemplate>
                    </Setter.Value>
                </Setter>
                <Setter Property="Template">
                    <Setter.Value>
                        <ControlTemplate TargetType="ListView">
                            <Border BorderBrush="{TemplateBinding BorderBrush}"
                            Background="{TemplateBinding Background}"
                            BorderThickness="{TemplateBinding BorderThickness}">
                                <ScrollViewer x:Name="ScrollViewer"
                                      TabNavigation="{TemplateBinding TabNavigation}"
                                      HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}"
                                      HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}"
                                      IsHorizontalScrollChainingEnabled="{TemplateBinding ScrollViewer.IsHorizontalScrollChainingEnabled}"
                                      VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}"
                                      VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
                                      IsVerticalScrollChainingEnabled="{TemplateBinding ScrollViewer.IsVerticalScrollChainingEnabled}"
                                      IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}"
                                      IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}"
                                      ZoomMode="{TemplateBinding ScrollViewer.ZoomMode}"
                                      IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}"
                                      BringIntoViewOnFocusChange="{TemplateBinding ScrollViewer.BringIntoViewOnFocusChange}"
                                      AutomationProperties.AccessibilityView="Raw">
                                    <ScrollViewer.TopLeftHeader>
                                        <Grid>
                                            <ContentPresenter Content="{Binding TopLeftContent}" 
                                                      BorderBrush="Transparent"
                                                      BorderThickness="0"
                                                      Padding="0"/>
                                        </Grid>
                                    </ScrollViewer.TopLeftHeader>
                                    <ScrollViewer.TopHeader>
                                        <Grid x:Name="PART_TopHeaderGrid">
                                            <ContentPresenter Content="{Binding TopContent}"
                                                      BorderBrush="Transparent"
                                                      BorderThickness="0"
                                                      Padding="0" />
                                        </Grid>
                                    </ScrollViewer.TopHeader>
                                    <ItemsPresenter x:Name="ScrollViewerItemsPresenter"
                                            Header="{TemplateBinding Header}"
                                            HeaderTemplate="{TemplateBinding HeaderTemplate}"
                                            HeaderTransitions="{TemplateBinding HeaderTransitions}"
                                            Footer="{TemplateBinding Footer}"
                                            FooterTemplate="{TemplateBinding FooterTemplate}"
                                            FooterTransitions="{TemplateBinding FooterTransitions}"
                                            Padding="{TemplateBinding Padding}"
                                            HorizontalAlignment="Left"
                                            VerticalAlignment="Top"/>
                                </ScrollViewer>
                            </Border>
                        </ControlTemplate>
                    </Setter.Value>
                </Setter>
            </Style>
        </Grid.Resources>

        <ListView x:Name="lv" Style="{StaticResource CustomListViewStyle}" ItemsSource="{Binding Items}" AutomationProperties.Name="Test ListView" Width="400" Height="400" Background="AliceBlue" HorizontalAlignment="Left" VerticalAlignment="Top" Margin="2"/>
    </Grid>

```
Code-behind:
```
namespace SVHeadersAccessibilityTest
{
    public sealed partial class MainPage : Page
    {
        private DataObj _dataObj = new DataObj();

        public MainPage()
        {
            this.InitializeComponent();

            this.lv.DataContext = _dataObj;
        }
    }

    public class DataObj
    {
        private List<string> _items = new List<string>();

        public DataObj()
        {
            _items.Add("String1");
            _items.Add("String2");
        }

        public string TopLeftContent
        {
            get
            {
                return "Top-Left-Content";
            }
        }

        public string TopContent
        {
            get
            {
                return "Top-Content";
            }
        }

        public List<string> Items
        {
            get
            {
                return _items;
            }
        }
    }
}

```

Build and run a C# UWP app with the code above. Inspect the automation tree with a tool like Accessibility Insights for Windows.
Result: the ScrollViewer headers are not visible.
Expect: the header should be visible like when the ScrollViewer is used outside the ListView control template.

This is a XAML framework issue that could be addressed in WinUI 3+.
